### PR TITLE
Fix-bug: datatable doesn't display expected 10 records when user clic…

### PR DESCRIPTION
**Pull request by dabeng**
Datatable doesn't display expected 10 records when user clicking search button

### CODE REVIEW

**Comment by ycheng-aa** [comment]
:+1:

File: src/app.jsx [hunk 1]
```diff
@@ -203,6 +203,7 @@ module.exports = React.createClass({
     if (data['role'] != 'all') {
       params['role'] = data['role'];
     }
+    params.page_size = 10;
```
Comment (simozhan) [inline src/app.jsx]: Is it better to set a global PAGE_SIZE=10, and others refers to it?

**Comment by simozhan** [comment]
:+1:

**Comment by ycheng-aa** [comment]
:+1:

**Pull request merged**